code/osapi/osapi.cpp

```diff
@@ -42,8 +42,21 @@ namespace
 		// Lazily initialize the preferences path
 		if (!preferencesPath) {
 		    preferencesPath = SDL_GetPrefPath(ORGANIZATION_NAME, APPLICATION_NAME);
-		    if (!preferencesPath) {
-			    mprintf(("Failed to get preferences path from SDL: %s\n", SDL_GetError()));
+			
+			// this section will at least tell the user if something is seriously wrong instead of just crashing without a message or debug log.
+			// It may crash later, especially when trying to load sound. But let's let it *try* to run in the current directory at least.
+		    if (preferencesPath == nullptr) {
+				static bool sdl_is_borked_warning = false;
+				if (!sdl_is_borked_warning) {
+					Warning(LOCATION, "%s\n\nFSO is having issues using SDL.dll, or it was not included. The build and dll may be in a protected folder,"
```

Comment:
This should be a `ReleaseWarning` so that it will show up for Release build.

Reply:
Changed, and I tried to make the message descriptive but also not try to do everything under the sun..